package.json

```diff
@@ -14,7 +14,7 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006 -c .storybook -s src/static",
     "build-storybook": "build-storybook -s ./src/static",
-    "test": "npm run test-lint && prettier --check --loglevel error './src/**/*' && node ./scripts/build.test.js && npm run test-jest && npm run test-chrome",
+    "test": "npm run test-lint && prettier --check './src/**/*' && node ./scripts/build.test.js && npm run test-jest && npm run test-chrome",
```

Comment:
@ REVIEWERS: minor bugfix - see logdetails if failing

Reply:
What else does this log? Doesn't this create a lot of noise in the console? We should see if this stage is failing, I agree, but if this introduces a lot of uninteresting logs, I would rather go with an approach like this:
`prettier --check --loglevel error './src/**/*' || echo 'Prettier failed' && exit 1` (in a separate npm script, like `npm run prettier`.

But anyway, you can decide yourself, just my 2 cents.

Reply:
I wanted to see the details why Prettier failed. And logs are logs. For me its ok that this is noisy. That should help me finding bugs faster.